### DISCUSSION TITLE
Reverse tunnel using openssh

### DISCUSSION THREAD

**Discussion by sghenimi** [gravitational/teleport#5687] [Q&A] [answered]
I need to have ssh access to devices that are behind firewall. The installation of teleport client ARMv7 version didn't succeed on the devices (due to the lib.so.6 librarie). I decided to use openssh to interact with my teleport server. My question is the following: is there any way to do reverse tunnel using openssh.

The configuration will be the following:

devices with openssh client behind firewall.
<----- Reverse tunnel -----
My server holding teleport server.

**Comment by webvictim** [comment, marked as answer]
Hey - this isn't currently possible. If you only need to run the node service on the ARM machine, you could try compiling OSS Teleport from source on your ARM box using the instructions here: https://github.com/gravitational/teleport/issues/4754#issuecomment-776222406

This would get around the glibc versioning issue (which I assume is the problem you're having?)

**Reply by sghenimi** [reply]
Yes, i compiled from source code and it 's working thank you

**Reply by webvictim** [reply]
Great news! We have [an open issue](https://github.com/gravitational/teleport/issues/5338) to look at reducing our glibc requirement to a lower version, but it's currently blocked due to other build constraints.

**Reply by sghenimi** [reply]
Great! wainting for that
